plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -148,9 +177,15 @@ def eval_isolation(self, port_name, cause):
         if port_name in self.ufm_latest_isolation_state:
```

Comment:
bad name for map or collection ... not meaningful

Reply:
ufm_latest_isolation_state? how would you call the latest state of isolated ports in UFM?